compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/UIKitTextInputService.uikit.kt

```diff
@@ -329,19 +329,21 @@ internal class UIKitTextInputService(
                 else -> UIReturnKeyType.UIReturnKeyDefault
             }
 
-        override fun textContentType(): UITextContentType? =
-            when (currentImeOptions?.keyboardType) {
-                KeyboardType.Password, KeyboardType.NumberPassword -> UITextContentTypePassword
-                KeyboardType.Email -> UITextContentTypeEmailAddress
-                KeyboardType.Phone -> UITextContentTypeTelephoneNumber
-                else -> null
-            }
-
-        override fun isSecureTextEntry(): Boolean =
-            when (currentImeOptions?.keyboardType) {
-                KeyboardType.Password, KeyboardType.NumberPassword -> true
-                else -> false
-            }
+        override fun textContentType(): UITextContentType? = null
```

Comment:
Do we need to comment both `textContentType` and `isSecureTextEntry`, or is it enough to comment only `isSecureTextEntry` to fix the issue with keychain?

I checked the example above on rc04, on simulator - keyboard changes correctly. If we comment `textContentType` - will keyboard be changing?

Unfortunately, I don't have a fast way to check the changes on iOS in the core repo right now.

Reply:
We need to comment them both. One of the returning value `true` or `UIKeyboardTypePasswrord` will break next usage of TextFields.